.github/workflows/test.yml

```diff
@@ -11,88 +11,72 @@ env:
 
 jobs:
 
-  lin:
-    strategy:
-        fail-fast: false
-        max-parallel: 3
-        matrix:
-          task: [
-            vhpidirect/quickstart/random,
-            vhpidirect/quickstart/math,
-            vhpidirect/quickstart/customc,
-            vhpidirect/quickstart/wrapping/basic,
-            vhpidirect/quickstart/wrapping/time,
-            vhpidirect/quickstart/linking/bind,
-            vhpidirect/quickstart/package,
-            vhpidirect/quickstart/sharedvar,
-            vhpidirect/shared/shlib,
-            vhpidirect/shared/dlopen,
-            vhpidirect/shared/shghdl,
-            vhpidirect/arrays/intvector,
-            vhpidirect/arrays/logicvector,
-            vhpidirect/arrays/matrices,
-            vhpidirect/arrays/matrices/framebuffer,
-            vpi/quickstart,
-          ]
+  lin-docker:
     runs-on: ubuntu-latest
     env:
       DOCKER_BUILDKIT: 1
-      DOCKER_IMAGE: ghdl/ghdl:buster-llvm-7
     steps:
     - uses: actions/checkout@v2
-    - run: docker pull $DOCKER_IMAGE
-    - run: docker run --rm -tv $(pwd):/src -e CI $DOCKER_IMAGE /src/${{ matrix.task}}/run.sh
+    - run: docker build -t cosim/test - < .github/Dockerfile
+    - run: docker run --rm -v $(pwd):/src -w /src -e CI cosim/test python3 -m pytest -v -s -ra test.py --color=yes
 
-  vunit:
-    strategy:
-        fail-fast: false
-        max-parallel: 3
-        matrix:
-          task: [
-            vhpidirect/arrays/matrices/vunit_axis_vcs,
-          ]
+  win-stable:
```

Comment:
The failure of this task might be legit. The `make` that is available in the PATH was not explicitly installed by us. It seems to be preinstalled from Chocolatey. We might investigate if we can make it work (e.g. `check_call(['make', 'run'], shell=True, cwd=str(self.vhpidirect / 'arrays' / 'intvector'))`) or whether we need to use `eine/setup-msys2` here too.

I'd say that v0.37 windows releases of GHDL do require a MINGW environment. However, it has been surprising for me that just extracting the zipfile was working fine until now. Maybe GHDL was picking the MINGW libs of Git for Windows, which is installed by default.

Reply:
Glad to see it worked. It would have been a pain to fix win-stable otherwise...